about/policy/open-sgid/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Open SGID Policy and Terms of Use
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+tags:
+  - policy
+  - sgid
+categories: []
+date: 2020-05-04 10:01:27 -0500
+---
+
+The following guidelines apply when using the Open SGID database. 
+
+## Do
+{: .text-left }
+
+- Search our [known-issues](https://github.com/agrc/open-sgid/issues) page if you are having trouble 
+- {% include contact.html subject=page.title text="Contact " contact=site.data.contacts.opensgid %} if you encounter an new issue
```

Comment:
```suggestion
- {% include contact.html subject=page.title text="Contact " contact=site.data.contacts.opensgid %} if you encounter a new issue
```

Reply:
good catch.  I guess I've committed since this suggestion - so i'll incorporate it into the next comment..